sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Utils.scala

```diff
@@ -134,7 +134,11 @@ private[sql] object DataSourceV2Utils extends Logging {
           None
         }
         val timeTravel = TimeTravelSpec.create(timeTravelTimestamp, timeTravelVersion, conf)
-        (CatalogV2Util.loadTable(catalog, ident, timeTravel).get, Some(catalog), Some(ident))
```

Comment:
Probably it's better to have a new method in CatalogV2Util that do the same as loadTable but without catch exceptions and return Table instead of Option[Table]

Reply:
Thanks, I have added a `getTable` method in `CatalogV2Util`.